### PR TITLE
Updates/javascript

### PR DESCRIPTION
**Pull request by maxcell**
Since @jaxbot can't do it himself but do not immediately merge without local tests.

### CODE REVIEW

**Comment by accell** [comment]
"The ice has a really interesting texture"

File: _src/workshops/13-javascript.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+}
+
+function bar() {
+  console.log(i);
+}
+
+foo();
+```
+
+Thee answer is 5, 2. A little strange, since we declared `i` in the outer scope, redeclared it in the inner scope of a function, then went to another context where the outer scope version was used, then back to one where our new scope was used.
```
Comment (maxcell) [inline _src/workshops/13-javascript.md]: "The"

**Comment by maxcell** [comment]
@jaxbot: I think that this is pretty good beyond that one spelling mishap. I got it working locally so it should be runnable and ready! :+1:

Be sure to have the workshop link on the front updated and it is :shipit:-worthy

**Pull request merged**